tools/cldr-apps/src/main/java/org/unicode/cldr/web/LocaleMaxSizer.java

```diff
@@ -0,0 +1,70 @@
+package org.unicode.cldr.web;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import org.unicode.cldr.util.CLDRLocale;
+
+/**
+ * This class tracks the expected maximum size of strings in the locale.
+ *
+ * @author srl
+ */
+public class LocaleMaxSizer {
```

Comment:
Question. By maximum size, do you mean the maximum **length** of any element value in the main locale file? It looks like the code assumes that, and also assumes that the longest one is the exemplar value in the CJK locales which is 8192, and that every other locale has a max of 4096. And we don't care that it is not exact numbers. If these are true, would be nice to add to the javadoc. (but not a blocker)

Reply:
Yes length

Reply:
Created a ticket https://unicode-org.atlassian.net/browse/CLDR-16777 to document.